### ISSUE TITLE
String representation of arrow functions

### ISSUE THREAD

**Issue by ningyuansg** [source-academy/cadet-frontend#102]
Current implementation:

```javascript
>>> const f = () => 0;
undefined
>>> f;
[object Object]
```

Expected implementation:

```javascript
>>> const f = () => 0;
undefined
>>> f;
() => 0
```

Or to similar effect.

**Comment by ningyuansg** [comment]
^ The behaviour above is already implemented. Issue is with representation of `es.CallExpressions` in instances of `MaximumStackLimtExceeded`. `[object Object]` comes from `node:es.CallExpression.arguments` which is an array of `es.Node` objects.

**Comment by remo5000** [comment]
Resolved in 01591c3
